### DISCUSSION TITLE
Density Cube File Boundaries

### DISCUSSION THREAD

**Discussion by sbembenek18** [grimme-lab/xtb#804] [Q&A] [answered]
How can I extend the grid boundaries for the density (cube file) calculation?

**Comment by MtoLStoN** [comment, marked as answer]
In fact, a manual extension of the grid boundaries is not possible with the current version of xTB. Based on your feedback, I just implemented the possibility of changing the default boundary offset with a detailed xtb input file (see PR #805). The input file (e.g. ``xtb.input``) should look like this:
```
$cube
   boff=3.0
```
You can then call xtb with ``xtb inp.xyz --input xtb.input``.  The default value for boff is 3.0. You can find more information about the [cube files](https://xtb-docs.readthedocs.io/en/latest/properties.html#cube-files) and about the [detailed xTB input](https://xtb-docs.readthedocs.io/en/latest/xcontrol.html) in our documentation.

This enhancement will be available in the next minor release. If you want to use it right away, I would suggest giving our [bleeding edge](https://github.com/grimme-lab/xtb/releases/tag/bleed) version a try, as soon as the PR is merged (which will be soon). Please note, however, that, while we try to keep everything stable at all times, this version is only minimally tested.

All the best

Marcel

**Reply by sbembenek18** [reply]
Thanks very much.

**Comment by sbembenek18** [comment]
I was looking to get version 6.6.1 but didn't see a conda install version
in conda-forge.

**Comment by sbembenek18** [comment]
Actually, the binary works perfectly! Look forward to using the adjustable
cube boundaries.
